avocado/plugins/messages.py

```diff
@@ -0,0 +1,123 @@
+# This program is free software; you can redistribute it and/or modify
```

Comment:
IIUC, this module it is essential to our runner. By removing this we would not have the runner working properly. Because of that, I would have chosen to put it inside `avocado.core.messages` instead of plugins. Any reason for using `plugins`  here?

Reply:
I used `plugins` here because the `messages` are used only to the `avocado.plugins.runner_nrunner` module. IIUC we should move `runner_nrunner` from `plugins` as well because is also essential to runner.